x-pack/test/observability_functional/apps/observability/pages/alerts/index.ts

```diff
@@ -85,6 +85,13 @@ export default ({ getService }: FtrProviderContext) => {
           await testSubjects.existOrFail('autocompleteSuggestion-field-kibana.alert.status-');
         });
 
+        it('Invalid input should not break the page', async () => {
```

Comment:
💬 This is the main test related to the bug. When testing locally, by disabling my fix, this test was failing.